server/eb/check_config.py

```diff
@@ -0,0 +1,39 @@
+import sys
+import argparse
+import yaml
+
+from server.common.config.app_config import AppConfig
+
+
+def main():
+    parser = argparse.ArgumentParser("A script to check hosted configuration files")
```

Comment:
Just curious -- is it possible to generalize this so that any config could be checked? We could move the logic into a separate class and then pull the checker in multiple locations -- for example here and in the CLI potentially for other folks who host.